react-native/react/native/listen-log-ui.js

```diff
@@ -3,8 +3,9 @@ import {logUiLog} from '../actions/notifications'
 
 export default function ListenLogUi () {
   engine.listenOnConnect('ListenLogUi', () => {
-    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}) => {
+    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}, response:any) => {
```

Comment:
Can't we just add a counter to preserve ordering (push that to the client side to deal with re-ordering)